### ISSUE TITLE
[stable-diffusion-x4-upscaler] Use pretrain VAE to encode a 512x512 image to latent space get nan, the image has been normalized to [-1,1]

### ISSUE THREAD

**Issue by leeruibin** [Stability-AI/stablediffusion#184]
I have downloaded the stable-diffusion-x4-upscaler pre-train model from https://huggingface.co/stabilityai/stable-diffusion-x4-upscaler

I try to fine-tune the upscaler model with my own data, however, I find when I encode the 512x512 image to latent space 128x128 with the pretrain VAE parameter, I get nan with size [b,4,128,128].

I have tracked the VAE forward function. I find that following the calculation map, the data will soon become huge and data overflow will happen.

![image](https://user-images.githubusercontent.com/26408630/221418245-8c19a5c6-8078-4aad-ac8b-0e6276902f7b.png)

I use the stable diffusion fine-tuning script in the following link and modify the script with my own dataset since there is no finetuning script for this x4-upscaler model.
https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/train_text_to_image.py

Is there any solution for this error?

**Comment by sczhou** [comment]
Hi @leeruibin , same here, did you solve this problem?

**Comment by vipzhe** [comment]
@leeruibin , I met same problem, did you know how to solve this problem?

**Comment by vipzhe** [comment]
> 
No half precision works

**Comment by Harper714** [comment]
Hi, any one fine-tuned the upscale model successfully?

**Comment by oubotong** [comment]
Same issue. Looking for some guidance on finetuning 4x upscale model

**Comment by Jamie725** [comment]
Just remove the half precision setting when loading the model, then it should work!
